### PR TITLE
TRUNK-4813:SQLException during the initial setup

### PR DESCRIPTION
**Pull request by mnsowmya**

### CODE REVIEW

File: api/src/main/resources/liquibase-update-to-latest.xml [hunk 1]
```diff
@@ -2355,10 +2355,13 @@
...
 		</comment>
 		<sql>
+		   <![CDATA[CREATE TEMPORARY table temp as(SELECT * from person_attribute_type);
+		    ]]>
 			<![CDATA[
-			UPDATE  person_attribute_type as A JOIN person_attribute_type as B ON A.name = B.name  SET A.sort_weight = (select count(*) from (select * from person_attribute_type) pat where pat.name < A.name) WHERE A.sort_weight is null;
+			UPDATE  person_attribute_type as A JOIN person_attribute_type as B ON A.name = B.name  SET A.sort_weight = (select count(*) from (select * from temp) pat where pat.name < A.name) WHERE A.sort_weight is null;
 			]]>
 		</sql>
+		<dropTable tableName="temp"/>
```
Comment (wluyima) [inline api/src/main/resources/liquibase-update-to-latest.xml]: Isn't the drop statement for a temporary table different? Or the regular drop works for temp tables too

Comment (MNSOWMYA) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @wluyima regular drop worked for me when i wrote <dropTemporaryTable tableName="temp"/>
it gave me error

**Comment by mnsowmya** [comment]
Closing this as i intend to create a new one with few more changes

File: api/src/main/resources/liquibase-update-to-latest.xml (hunk 1 above)
Comment (wluyima) [inline api/src/main/resources/liquibase-update-to-latest.xml]: I would assume liquibase doesn't have an equivalent tag, you might have raw SQL, what error did you get?

Comment (mnsowmya) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @wluyima    Invalid content was found starting with element 'dropTemporaryTable' .This was the error

Comment (wluyima) [inline api/src/main/resources/liquibase-update-to-latest.xml]: Did you execute raw SQL or some liquibase tag?

Comment (mnsowmya) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @wluyima  no i did not.I wrote it in the tag for my query it showed me line number.

Comment (mnsowmya) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @wluyima The error was due to the liquibase tag i used

Comment (wluyima) [inline api/src/main/resources/liquibase-update-to-latest.xml]: Which tag?

Comment (mnsowmya) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @wluyima <dropTemporaryTable tableName="temp" />

Comment (wluyima) [inline api/src/main/resources/liquibase-update-to-latest.xml]: Which error? Can you share the stacktrace?

**Pull request closed without merging**